README.md

```diff
@@ -78,6 +78,23 @@ The `mount_point` specifies the directory on which the file system will be mount
 ##### `mount_options`
 The `mount_options` specifies custom mount options as a string, e.g.: 'ro'.
 
+##### `raid_level`
+Specifies RAID level when type is `raid`.
+Accepted values are: `linear`, `striped`, `raid0`, `raid1`, `raid4`, `raid5`, `raid6`, `raid10`
+
+#### `raid_device_count`
+When type is `raid` specifies number of active RAID devices.
+
+#### `raid_spare_count`
+When type is `raid` specifies number of spare RAID devices.
+
+#### `raid_metadata_version`
+When type is `raid` specifies RAID metadata version as a string, e.g.: '1.0'.
+
+#### `raid_chunk_size`
+When type is `raid` specifies RAID chunk size as a string, e.g.: '512 KiB'.
+Chunk size has to be multiple of 4 KiB.
+
```

Comment:
Are those parameters accepted for RAID pools as well? They are not described there. (If yes, maybe it would be enough to describe them once and cross-reference.)

Reply:
^ @dwlehman

Reply:
Pools only accept `raid_level` as of now, and it has a different default for pools (`null`) than for raid volume (`raid0` IIRC).